### DISCUSSION TITLE
How to use  content picker field in where or order of GraphQL

### DISCUSSION THREAD

**Discussion by happygb21** [OrchardCMS/OrchardCore#15881] [Q&A]
![image](https://user-images.githubusercontent.com/34257310/56479873-2d800400-64ea-11e9-95b8-239f509e5437.png)
example：
{
   case(where:{blogPost:{contentItemId:""}}){
      displayText
   }
}

**Comment by chinasqzl** [comment]
I don't know if that's what you want.

https://orchardcore.readthedocs.io/en/dev/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/#querying-related-content-items

{ 
  person  { 
    name 
    favoriteMovies  { 
      contentItems  { 
        ...  on  Movie  { 
          name 
          releaseYear 
        } 
      } 
    } 
  } 
}

**Comment by happygb21** [comment]
@chinasqzl  Thank you, but what I want is to include content picker field in Graphql's query condition and query it, which bothers me a lot.

**Comment by zl2fxy** [comment]
Field filtering does not seem to be supported，I  am not found anything about field filtering at the graphql module.
if you find it,please tell me,thanks

**Comment by sebastienros** [comment]
It could be a separate query that targets the index that this field is generating. But I don't think we have that already.

Another option is to expose a custom SQL query in graphql, which is already support AFAIK.

**Comment by zl2fxy** [comment]
> It could be a separate query that targets the index that this field is generating. But I don't think we have that already.
> 
> Another option is to expose a custom SQL query in graphql, which is already support AFAIK.

OK，I think it's good，i in the process of finding the achieving graphql query filter about the field，
can you give me a simple example？thanks！

**Comment by happygb21** [comment]
@sebastienros  thanks~!
